### PR TITLE
Bug - parisma-access-logout-user domain arg is not required 

### PR DESCRIPTION
**Pull request by MLainer1**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-19141)

## Description
In the discussion [here](https://jira-hq.paloaltonetworks.local/browse/CYR-24735) the prisma access team confirmed that for some users, when stating the domain arg along with the request, the command does not work (the logout does not happen).

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PrismaAccess/ReleaseNotes/1_0_13.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Prisma Access
+- Updated the Docker image to: *demisto/netmiko:1.0.0.46467*.
+- Fixed an issue where the command **prisma-access-logout-user** did not work properly when stating the *domain* argument.
```
Comment (JudahSchwartz) [inline Packs/PrismaAccess/ReleaseNotes/1_0_13.md:5]:
```suggestion
- Fixed an issue where the command **prisma-access-logout-user** did not work properly when the *domain* argument was provided. 
```

File: Packs/PrismaAccess/Integrations/PrismaAccess/PrismaAccess.py [hunk 2]
```diff
@@ -397,12 +397,19 @@ def prisma_access_logout_user(computer: str, domain: str, user: str) -> Dict[str
     if apiConfigured:
         xmlComputer = '<computer>%s</computer>' % b64encode(computer.encode('utf8')).decode('utf8') if computer else ''
         b64User = (b64encode(user.encode('utf8'))).decode('utf8')
+        cmd = ''
+        if domain:
+            cmd = '''<request><plugins><cloud_services><gpcs>
```
[the remark this answers is not in the document]
Comment (MLainer1) [reply Packs/PrismaAccess/Integrations/PrismaAccess/PrismaAccess.py:402]: Wanted to match the rest of the code in the file, can change if you think it is necessary

File: Packs/PrismaAccess/Integrations/PrismaAccess/PrismaAccess.yml [hunk 3]
```diff
@@ -56,8 +56,8 @@ script:
       default: true
       description: Username to logout. (Without domain name - e.g. jsmith)
     - name: domain
-      required: true
-      description: Domain name of the user to logout.
+      required: false
+      description: Domain name of the user to logout. For some users, this field is requierd.
```
[the remark this answers is not in the document]
Comment (MLainer1) [reply Packs/PrismaAccess/Integrations/PrismaAccess/PrismaAccess.yml:60]: Some users does not have a domain attached to them

**Comment by MLainer1** [comment]
This PR needs a force merge due to not having an instance and not having a test-playbook
Screen Shot 2023-02-05 at 14 10 55

File: Packs/PrismaAccess/Integrations/PrismaAccess/PrismaAccess.yml (hunk 3 above)
Comment (JudahSchwartz) [inline Packs/PrismaAccess/Integrations/PrismaAccess/PrismaAccess.yml:60]:
```suggestion
      description: Domain name of the user to logout. For some users, this field is required.
```

File: Packs/PrismaAccess/ReleaseNotes/1_0_13.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+##### Prisma Access
+- Updated the Docker image to: *demisto/netmiko:1.0.0.47037*.
+- Fixed an issue where the command **prisma-access-logout-user** did not work properly when the *domain* argument was provided.
```
Comment (JudahSchwartz) [inline Packs/PrismaAccess/ReleaseNotes/1_0_13.md:5]:
```suggestion
- Fixed an issue where the command **prisma-access-logout-user** did not work properly when the *domain* argument was provided unnecessarily. 
```

**Review by JudahSchwartz — APPROVED** [review]
Note the doc changes

**Pull request merged**